gm4_holographic_tags/pack.mcmeta

```diff
@@ -0,0 +1,22 @@
+{
+  "pack":{
+    "pack_format":0,
+    "description":"A Gamemode 4 Module"
+  },
+  "module_name":"Holographic Tags",
+  "module_id":"holographic_tags",
+  "site_description":"Setup Floating Messages with a simple name tag!",
```

Comment:
should be "Set Up"

Reply:
The capitalisation seems a bit off as well, might I add